.github/workflows/ci.yml

```diff
@@ -149,6 +149,15 @@ jobs:
           args: --debug --path ./forc-fmt
       - name: Initialize test project
         run: forc init test-proj
+      - name: Update project manifest to use local sway-lib-std
+        run: echo '[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "test-proj"
+
+[dependencies]
+std = { path = "../../sway-lib-std" }' > ./Forc.toml
```

Comment:
Hmmm. Rather than nuking the entire manifest, I would prefer just appending the local `std` dep: https://github.com/FuelLabs/sway/issues/1458#issuecomment-1116162972

Reply:
I agree.. that minimizes the need to change this `echo` when `forc init` changes in the future.